src/features/todos.ts

```diff
@@ -2,7 +1,21 @@
+import { createSelector, createSlice } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
+import { RootState } from '../app/store';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set: (_, action) => {
+      return action.payload;
+    },
+  },
 });
+
+const todos = (state: RootState) => state.todos;
```

Comment:
The selector 'todos' assumes that 'todos' is a property on the RootState. Ensure that the RootState interface/type includes a 'todos' property that matches the expected state shape.